requirements.txt

```
@@ -1,6 +1,5 @@
-hikari[speedups]~=2.0.0.dev101; python_version < "3.10"
-hikari~=2.0.0.dev101; python_version == "3.10"
-hikari-lightbulb~=2.0.0
-git+https://github.com/FasterSpeeding/Tanjun@c1cb6c4aa1b103cda92f6779753867102ceb12d9
+hikari[speedups]~=2.0.0.dev112
 hikari-crescent>=0.3.3,<1.0
+hikari-lightbulb~=2.2.0
+hikari-tanjun~=2.9.0
```

```suggestion
hikari-lightbulb>=2.2.0,<3.0
hikari-tanjun>=2.9.0,<3.0
```
It may be worth opening these constraints up to accept future updates like Crescent does.